### PR TITLE
Added VGG implementation.

### PR DESCRIPTION
**Pull request by burntcarrot**
Added implementation of VGG using PyTorch, generic class is used for generating all types of VGG including:
- VGG-11
- VGG-13
- VGG-16
- VGG-19

### CODE REVIEW

**Comment by oke-aditya** [comment]
Looks good, though I need slightly deeper look once.

**Comment by burntcarrot** [comment]
@oke-aditya Have you checked it yet?

**Comment by oke-aditya** [comment]
Not yet, don't worry. I will review it tomorrow. I have been really busy today 🙁

**Comment by oke-aditya** [comment]
There are actually two cases of VGG one with batch_norm and one without. Though this seems for without batchnorm, and it is correct as well.

I guess this is good though

**Review by oke-aditya — APPROVED** [review]
